fseai/vision.py

```diff
@@ -0,0 +1,129 @@
+"""
+Computer Vision Packages
+"""
+import torch
+from torchvision import transforms, datasets
+from torch.utils.data import DataLoader
+import torch.nn as nn
+from torch import optim
+from FSEai import fseai_models as models
+
+
+def input_mnist():
+    """
+    Function to load mnist dataset
+    """
+    trainset = datasets.MNIST('', download=True, train=True, transform=transforms.ToTensor())
+    testset = datasets.MNIST('', download=True, train=False, transform=transforms.ToTensor())
+
+    print('MNIST train and test data set Loaded!')
+    return trainset, testset
+
+
+def build_mnist_model():
+    """
+    Function to build a fully connected model 
+    :return: model
+    """
+    # Import the Fully Connected network model from fseai_models
+    model = models.FCmodel
+    input_size = 784
+    hidden_size = [128, 64]
+    output_size = 10
+    model = model(input_size, hidden_size, output_size)
+    return model
+
+
+def train_mnist(model, trainset):
+    """
+    Function to train mnist data with Fully Connected model
+    """
+    global correct, total
```

Comment:
You don't seem to be using these anywhere else. Why are they declared in global scope? (Keep in mind that global variables are generally considered as bad programming practice, so don't use them unless it's necessary)